### DISCUSSION TITLE
Cannot connect to MSSQL server v2012 Express if using SSL

### DISCUSSION THREAD

**Discussion by p1nkv4lkyr13** [requarks/wiki#3266] [Error / Bug Report]
**Description**
My Wiki app cannot connect to a remote working MSSQL server. I did checked the connection by SSMS and it's fine.
Below is my db config:
```
db:
  type: mssql
  host: 192.168.68.223
  port: 1433
  user: username
  pass: password
  db: mydb
  ssl: true
```

**Error message**
`2020-12-02T03:12:04.265Z [MASTER] error: Database Initialization Error: Failed to connect to 192.168.68.223:1433 - 5672:error:1425F102:SSL routines:ssl_choose_client_version:unsupported protocol:c:\ws\deps\openssl\openssl\ssl\statem\statem_lib.c:1942:`

**Expected behavior**
Wiki app can connect to MSSQL Server with SSL.

**Host Info:**
 - OS: Windows Server 2019 Standard
 - Wiki.js version: 2.5.170
 - Node version: 14.15.1
 - MSSQL server version: 2012 Express

**Additional context**
I found a bug in the MSSQL configuration.
I tried not to use SSL by set ssl to false but still got the same error message.
Then I checked the _**/server/core/db.js**_ file and changed the ssl configuration as below:

_from this:_
```
case 'mssql':
...
if (dbUseSSL) {
    dbConfig.encrypt = true
    _.set(dbConfig, 'options.encrypt', true)
}
```
_to this:_
```
case 'mssql':
...
if (dbUseSSL) {
    dbConfig.encrypt = true
    _.set(dbConfig, 'options.encrypt', true)
} else {
    dbConfig.encrypt = false
    _.set(dbConfig, 'options.encrypt', false)
}
```
Now app can start successfully with no-ssl. But I still want to use encrypted connection.
Hope you guys can help. Thanks.

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)
